docs/MediaCodecTests.md

```diff
@@ -0,0 +1,90 @@
+# How to use MediaCodec Tests
+
+## Background
+
+In Android Media, there is a codec framework team and a video codec quality team. There was a gap in testing for audio codec quality. We wanted to create a tool to find and reproduce audio codec quality issues.
```

Comment:
For Markdown, a paragraph can span multiple lines of input and it will get wrapped into a single paragraph of output.